qa/suites/upgrade/nautilus-x/parallel/5-final-workload/dashboard.yaml

```diff
@@ -0,0 +1,28 @@
+tasks:
+  - rgw: [client.1]
+  - print: "**** done running tasks.mgr.test_* tests start"
+  - cephfs_test_runner:
+      fail_on_skip: false
+      modules:
+        - tasks.mgr.test_dashboard
```

Comment:
How do we make sure new tests added to the original dashboard.yaml are also added in the upgrade test?

Reply:
perhaps add a comment in the original dashboard.yaml and this one to note that they should be kept in sync

Reply:
is there some wildcard we could use with cephfs_test_runner instead of specifying every one?

Reply:
wildcard would be very good ability !

Reply:
+1 for the wildcard!